server/app/services/applicant/question/Scalar.java

```diff
@@ -37,6 +37,7 @@ public enum Scalar {
   SELECTIONS("selections", ScalarType.LIST_OF_STRINGS),
   STATE("state", ScalarType.STRING),
   STREET("street", ScalarType.STRING),
+  SUFFIX("suffix", ScalarType.STRING),
```

Comment:
**nitpick:** Same here, my preference would be to add the word "name".  So `NAME_SUFFIX`.

Reply:
+1. I think when it's clear the `suffix` is part of a name, like in a name question in the API, we don't need to say `name_suffix`, but in lists like this, where it's a lot of concepts mixed together, it's helpful to include `name`.

Reply:
Changing the scalar to `NAME_SUFFIX` will also end up changing the CSV header, so you'll need to manually set it to `suffix` the way we change the currency header, in https://github.com/civiform/civiform/blob/bad18e641f95f7cbd90a699ba7a259af2ab4be93/server/app/services/export/CsvExporterService.java#L303